.gitignore

```diff
@@ -10,3 +10,4 @@
 /php-cs-fixer.phar.asc
 /phpunit.xml
 /vendor/
+/.idea/
```

Comment:
Can you share your reasoning why adding this entry, please?

Reply:
Entries that are specific to contributors and not to the repository itself should not be added here. Please add them to `.git/info/exclude` or a [global ignore file](https://git-scm.com/docs/gitignore).

Reply:
Thank you for letting me learn another trick.